scripts/release.js

```diff
@@ -178,7 +188,13 @@ if (release_option === '5') {
 } else {
   spawnOrFail('git', ['push origin HEAD:release -f']);
 }
-process.chdir(path.join(__dirname, '../demo/meeting/serverless'));
-logger.log("Deploying unique release candidate Meeting Demo URL...");
-const formattedVersion = versionString.replace(/\./g, "-");
-spawnOrFail('node', [`./deploy.js -r us-east-1 -b chime-sdk-components-demo-${formattedVersion} -s chime-sdk-components-demo-${formattedVersion}`]);
+
+spawnOrFail('git', [
+  'clone https://github.com/aws-samples/amazon-chime-sdk.git',
+]);
+process.chdir(path.join(__dirname, '../apps/meeting/serverless'));
```

Comment:
Have you tested this? You need to update it to the new code to GitHub master.

Reply:
I run `npm run build:release`, is it the right way to test?

Reply:
Nope. it should be npm run release to actually trigger release.js (see package.json).